src/store/ChartStore.js

```diff
@@ -562,6 +562,10 @@ class ChartStore {
         }
     };
 
+    getMaxMasterDataSize(granularity) {
+        return granularity === 86400 ? Math.floor(2.8 * 365) : 5000;
```

Comment:
@nazaninreihani how about 8 hours granularity? the number of ticks is less than 5000 for 3 years.

Reply:
oh okay I'll add that too